AMS won't compile

Arduino: 1.6.4 (Mac OS X), Platine: "Arduino Uno"

Build-Optionen wurden verändert, alles wird neu gebaut
Verwende die Bibliothek SPI im Ordner: /Applications/Arduino.app/Contents/Java/hardware/arduino/avr/libraries/SPI 
Verwende die Bibliothek AFMotorDrawbot im Ordner: /Users/jan/Documents/Arduino/libraries/AFMotorDrawbot (legacy)
Verwende die Bibliothek Wire im Ordner: /Applications/Arduino.app/Contents/Java/hardware/arduino/avr/libraries/Wire 
Verwende die Bibliothek Adafruit_MotorShield im Ordner: /Users/jan/Documents/Arduino/libraries/Adafruit_MotorShield (legacy)
Verwende die Bibliothek Servo im Ordner: /Applications/Arduino.app/Contents/Java/libraries/Servo 
Verwende die Bibliothek SD im Ordner: /Applications/Arduino.app/Contents/Java/libraries/SD 
Verwende die Bibliothek EEPROM im Ordner: /Applications/Arduino.app/Contents/Java/hardware/arduino/avr/libraries/EEPROM 

/Users/jan/Documents/Arduino/libraries/AFMotorDrawbot/AFMotorDrawbot.cpp: In member function 'void AF_Stepper::onestep(uint8_t)':
/Users/jan/Documents/Arduino/libraries/AFMotorDrawbot/AFMotorDrawbot.cpp:256:10: error: return-statement with a value, in function returning 'void' [-fpermissive]
   return currentstep;
          ^

--- a/firmware_ams/firmware_ams.ino
+++ b/firmware_ams/firmware_ams.ino
@@ -145,6 +145,8 @@
 
 #include "Vector3.h"
 
+// V1 Motor Shield https://learn.adafruit.com/adafruit-motor-shield/library-install
+#include <AFMotor.h>
 
 //------------------------------------------------------------------------------
 // VARIABLES

